Automattic/themes
# Modern Business: Refactor the Full Site Editing styles

#### Changes proposed in this Pull Request

Requires https://github.com/Automattic/wp-calypso/pull/35387

Refactor the full Editing Styles in the Modern Business theme.

Before/after screenshots can be found in https://github.com/Automattic/wp-calypso/pull/35387

#### Testing instructions



* Build FSE with https://github.com/Automattic/wp-calypso/pull/35387.
* On an FSE site, edit a page.
* Try poking the header and footer template parts, and make sure that:
  * The Template block overlay is properly aligned to the editor frame.
  * The overlay retains the same background color and blur filter as before.
  * Selecting (clicking) a Template block does not close the sidebar (if it was open), but simply hides its "Block" tab.
  * Everything looks fine!
